Etiquette/CODE_REVIEW.md

```diff
@@ -0,0 +1,108 @@
+* Approve changes if you are confident
+* Comment on the changes if something is not clear to you and needs further clarification or actions
+* If you are not familiar with the domain ask for more context (better in person) rather than skipping to something more familiar
+* Ask, don't tell. ("What do you think about...?", "How about ...?", "Should we try ...?")
+* Be breif but clear in your comments (code review is not about proving who is more correct or knows more)
+* Unless explicitly asked for it do not suggest code to be written in a specific way - let the author to find or suggest a solution themselves, they may come up with a better one
+* Submit all the comments as a single review, even if there is just one comment
+* If you see a stale pull request without any updates - ask for the status update in the comment
+* Do not be a blocker - if you requested changes and they were addressed make sure you review it
+* Do not merge others pull requests - they may be waiting for some one else review
+
+Responding to feedback
+-------------------------
+
+* Wait before answering to the single comments, reviewer may have more (though they should prefer to send all comments at once)
+* Be grateful for the reviewer's suggestions. ("Good call. I'll make that
+  change.")
+* If you disagree with suggestion explain way and avoid template answers like "out of the scope", "thank you for your opinion". If you think that suggestion is a personal preference and does not bring value - say so
+* Assume the best intention from the reviewer's comments.
+* If a reviewer seems aggressive or angry or otherwise personal, consider if it is intended to be read that way and ask the person for clarification of intent, in person if possible. Do not escalate online discussions.
+* If you can't reach agreement seek for more opinions
+* Don't be too defensive about your code, someone will rewrite it anyway sooner or later (it might be yourself as well)
+* Try to answer all comments. You can review your own code for that instead of sending individual replies
+* Add "work in progress" label while addressing review comments
+* Re-request review when you addressed the comments (either implementing suggested change or replying to the comment) and label it with "Ready for review" again
+* Do not rebase and force-push in your pull request branch after you recieved any comments, use merge intsead. All pull requests are squashed when they are merged into the base branch so the history of your changes won't matter. It's fine to rebase and force-push if there are no comments
+* Do no merge pull request with pending requests for changes unless you agreed with their authors that it was addressed
+* When you have two approvals and don't wait for review from anyone else - put a "Merge" label for bot to merge your changes.
+
+
+PR Parties
+--------
+
+To improve colloboration every week (on Monday) we take time to review pull request together in small groups, what we call "PR parties"
```

Comment:
```suggestion
To improve collaboration every week (on Monday) we take time to review pull request together in small groups, what we call "PR parties".
```

Reply:
I first read that as "to improve collaboration every week, ..." so took me a second to get the meaning of this. Maybe rewriting it like this would help?
```suggestion
To improve collaboration, we take time to review pull request together in small groups every week (on Monday). This is what we call "PR parties".
```